File: seongwon/Programmers/Level_0/Lv0_특별한_이차원_배열_2.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<vector<int>> arr) {
    int answer = 1;
    for(int i = 0; i < arr.size(); i++){
        for(int j = 0; j < arr.size(); j++){
            if(arr[i][j] != arr[j][i]){
                answer = 0;
                return answer;
            }
        }
    }
    return answer;
}

File: dohyeon/Programmers/Level_0/Lv0_특별한_이차원_배열_2.py
def solution(arr) :
    answer = 1
    for i in range(len(arr)) :
        for j in range(len(arr[i])) :
            if arr[i][j] != arr[j][i] : return 0
    return answer

print(solution([[5, 192, 33], [192, 72, 95], [33, 95, 999]])) # 1